pkg/scheduler/api/job_info.go

```diff
@@ -365,12 +365,20 @@ func (ji *JobInfo) SetPodGroup(pg *PodGroup) {
 	ji.CreationTimestamp = pg.GetCreationTimestamp()
 
 	var err error
-	ji.WaitingTime, err = ji.extractWaitingTime(pg)
+	ji.WaitingTime, err = ji.extractWaitingTime(pg, v1beta1.JobWaitingTime)
 	if err != nil {
 		klog.Warningf("Error occurs in parsing waiting time for job <%s/%s>, err: %s.",
 			pg.Namespace, pg.Name, err.Error())
 		ji.WaitingTime = nil
 	}
+	if ji.WaitingTime == nil {
+		ji.WaitingTime, err = ji.extractWaitingTime(pg, JobWaitingTime)
```

Comment:
why we have different keys `JobWaitingTime` and `v1beta1.JobWaitingTime` here?

Reply:
The old key `JobWaitingTime` is only available to volcano jobs, whereas new key `v1beta1.JobWaitingTime` can be used in both volcano job and k8s workloads. So new key can substitute the old one basically.

The old key was reserved temporarily to ensure that previous users still could use SLA plugin as usual. After all annotation-keys were available, a doc would be proposed to introduce these keys, and then old keys could be removed.